spec.bs

```diff
@@ -2792,6 +2834,23 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+The {{Fence/notifyEvent}} method of the {{Fence}} interface introduces a new event type, <code><dfn>fencedtreeclick</dfn></code>.
```

Comment:
This method does not introduce this new event type. This spec does; but any code create events of this type. I would dissociate these two.

Reply:
I removed this line.